pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -654,6 +704,35 @@ MetricsConfiguration metricsConfiguration() {
           "--metrics-enabled option and --metrics-push-enabled option can't be used at the same "
               + "time.  Please refer to CLI reference for more details about this constraint.");
     }
+    
+    CommandLineUtils.checkOptionDependencies(
+        logger,
+        commandLine,
+        "--metrics-enabled",
+        !isMetricsEnabled,
+        Arrays.asList(
+            "--metrics-host",
+            "--metrics-port"));
+            
+    CommandLineUtils.checkOptionDependencies(
+        logger,
+        commandLine,
+        "--metrics-push-enabled",
+        !isMetricsPushEnabled,
+        Arrays.asList(
+            "--metrics-push-host",
+            "--metrics-push-port",
+            "--metrics-push-interval",
+            "--metrics-push-prometheus-job"));
+        logger,
```

Comment:
github suggestions left in lines 727 to 735.   Looks like they don't support multiline suggestions or I did my comment wrong.

Reply:
I fixed it by hand but yes, I think it means that it's single line only.